Framework/include/ScaleFactors.h

```diff
@@ -478,16 +478,18 @@ class ScaleFactors
                 const double muTotSFPErr2       = muNoTrigSFPErr2 + muTrigSFPErr*muTrigSFPErr;
                 const double muNonIsoTotSFPErr2 = muNoTrigSFPErr2 + muNonIsoTrigSFPErr*muNonIsoTrigSFPErr;
                 
```

Comment:
I guess if this is left out for now, can you put in some comment about why commented out.

Reply:
Added comment, will follow up with the SUSY Muon contact to make sure this is okay